tools/links/links.xml

```diff
@@ -0,0 +1,202 @@
+<tool id="links" name="LINKS" version="@TOOL_VERSION@+galaxy+@VERSION_SUFFIX@" python_template_version="3.5" profile="20.09">
+    <description> - scaffold genome assemblies with long reads</description>
+    <macros>
+        <token name="@TOOL_VERSION@">2.0.1</token>
+        <token name="@VERSION_SUFFIX@">1</token>
+    </macros>
+    <xrefs>
+        <xref type="bio.tools">links</xref>
+    </xrefs>
+    <requirements>
+        <!-- Bioconda containers
+        (Set job runner to singularity to use biocontainers)
+        2.0.1 is missing LINKS_CPP
+        2.0.0 is missing perl -->
+        <!-- <requirement type="package" version="@TOOL_VERSION@">links</requirement>  -->
+        <!-- Working container -->
+        <container type="docker">ghcr.io/tomharrop/container-links:v@TOOL_VERSION@-@VERSION_SUFFIX@</container>
```

Comment:
Can you update this to a SHA checksum? Like `<container type="docker">neoformit/alphafold-galaxy@sha256:6adf7f07062b307d08c11130c39a28abc7c290b23f6c347b09c2c649c054c338</container>`

Reply:
Adding the digest after the tag will break the singularity runner
```
unable to parse image name docker://ghcr.io/tomharrop/container-links:v2.0.1-1@sha256:5f22be0d47df2ada61056fc4cbad1cf17fe03748d3a55201eb72dba5a6eb35e2: Docker references with both a tag and digest are currently not supported
```
I've done it with *just* the digest, but I don't know if the Docker runner will like that
```
ghcr.io/tomharrop/container-links@sha256:5f22be0d47df2ada61056fc4cbad1cf17fe03748d3a55201eb72dba5a6eb35e2
```

Reply:
@TomHarrop is there a way to fix the bioconds recipe instead?

Reply:
Thanks @bgruening, I'll raise it with the LINKS developer since conda seems to be preferred.

I'll take a look at the recipe but I don't use conda myself, i.e. I may not be in a position to fix it.

Reply:
@bgruening you may remember that I encountered a [world of issues](https://github.com/bioconda/bioconda-recipes/issues/32088) last month trying to add a dependancy to the `smudgeplot` recipe. I still haven't been able to install the updated package in Galaxy with Conda.

Reply:
Thanks all, I made a PR and tagged the LINKS maintainer.

Not sure if my fix is the right way to do it, and I can't test the biocontainer that is built from the conda package, so it will be up to them from here.

https://github.com/bioconda/bioconda-recipes/pull/33166

Reply:
So with this new bioconda package we can drop the explicit container and use the normal `package` isn't it?

Reply:
Yep! It’s on its way.